lib/puppetserver/ca/action/generate.rb

```diff
@@ -77,31 +77,34 @@ def run(input)
           end
 
           # Load, resolve, and validate puppet config settings
-          puppet = Config::Puppet.parse(config_path)
+          settings_overrides = {}
+          settings_overrides[:certname] = input['certname'] unless input['certname'].empty?
```

Comment:
Do we want to add ca-name and certname to the USAGE above too?

Reply:
Sure. Seems like maybe instead of a new line for each flag, we should just list them all together. What do you think? I feel like that's more common for usage banners unless the options are mutually exclusive.

Reply:
Yeah that's probably fine.  I'm no usage banner expert ;)